readme.md

```diff
@@ -10,6 +10,21 @@ Package maintainers will be able to publish JavaScript files to NPM in this form
 
 Do not confuse this format with an AST. It is designed purely for interoperability and publishing. Using this format you can build an AST or regular CSS or any other structure.
 
+### Glossary
+
+The following are some terms that are used throughout the spec, or are relevant when discussing the spec.
+
+- **Markers:** Like nodes in an AST format, these tuples indicate significant literals and values in CSSinJS (See [Markers](#markers))
```

Comment:
mb not just CSSinJS, right? I am thinking about future support of sass and stylus

Reply:
I used it because it's being referenced above, but I can remove it and the description will still make sense, I suppose